invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/views/card.variables

```diff
@@ -0,0 +1,2 @@
+@largeDesktopScreenCardHeight: 305px;
```

Comment:
Yes but they are really custom values because the width is being calculated dynamically for the cards. Maybe it would be worth to have a custom css for square cards instead of using SUI `.five.cards`. I also tried to look for a SUI variable to use, instead of creating new ones but couldn't find ....